### PR TITLE
chore: update more dependencies

### PR DESCRIPTION
**Pull request by krzysu**
let's see if everything still works

### CODE REVIEW

File: examples/web-wagmi/package.json [hunk 1]
```diff
@@ -26,19 +26,20 @@
     "react-hot-toast": "^2.4.0",
     "react-router-dom": "^6.8.1",
     "readable-web-to-node-stream": "^3.0.2",
-    "wagmi": "0.9.2"
+    "wagmi": "^0.11.6"
```
Comment (cesarenaldi) [inline examples/web-wagmi/package.json:29]: I think we need to be careful with wagmi. It's in initial development phase (`0.x`) and they definitely make breaking changes, we experiences one ourself between `0.8.x` and `0.9.x`. Given there is no code changes required on our side between `0.9.2` and `0.11.6` I would consider an range: `">=0.9.2 <=0.11.6"`.

@desfero what do you think?

Comment (desfero) [reply examples/web-wagmi/package.json:29]: agree, we need to narrow the `wagmi` to a specific range that we know works properly inside `@lens-sdk/wagmi` `peerDependencies` configuration.
We probably needs some changeset that's mentioning which important deps were updated

File: packages/wagmi/package.json [hunk 2]
```diff
@@ -44,12 +44,12 @@
...
     "react-dom": "^18.2.0",
     "typescript": "^4.9.5",
-    "wagmi": "^0.11.6"
+    "wagmi": ">=0.9.2 <=0.11.6"
   },
   "peerDependencies": {
     "@lens-protocol/react": "workspace:*",
     "ethers": "^5.7.2",
-    "wagmi": "^0.11.5"
+    "wagmi": ">=0.9.2 <=0.11.6"
```
Comment (desfero) [inline packages/wagmi/package.json:52]: We should have the range only here inside `peerDependencies`, all the other places are considered internal and can use specific versions (with or without `^` prefix).

Comment (krzysu) [reply packages/wagmi/package.json:52]: ok 👌

**Review by cesarenaldi — APPROVED** [review]
Thank you for looking into this and follow up on review comments.

**Pull request merged**